operations/mimir-tests/test-deployment-mode-migration-generated.yaml

```diff
@@ -2401,6 +2401,7 @@ spec:
         - -server.grpc.keepalive.min-time-between-pings=10s
         - -server.grpc.keepalive.ping-without-stream-allowed=true
         - -server.http-listen-port=8080
+        - -store-gateway.sharding-ring.auto-forget-enabled=false
```

Comment:
This is interesting. We should not apply the change while the migration is in progress.